### ISSUE TITLE
Improve star names in  Almagest SC

### ISSUE THREAD

**Issue by abalkin** [Stellarium/stellarium#1675]
> While I disagree with some choices, changing existing names is outside of the scope of this PR. I will open separate issues to discuss possible changes.

Thank you! 
I will be happy to discuss the translations: we already discussed this at several occations but not as consequently as I would wish to. It always helps to develop a common sense on how to express something in words.

_Originally posted by @sushoff in https://github.com/Stellarium/stellarium/pull/1647#issuecomment-842396604_

**Comment by abalkin** [comment]
@sushoff - the main issue that I would like to discuss here is the use of Greek script when English is selected as the sky culture language.  While I realize that (Ancient?) Greek spelling is the most authentic, it is more user friendly to provide translatable phonetic transliterations.  For  advanced users we can provide a choice of Ancient Greek in the sky culture language menu.

**Comment by sushoff** [comment]
I agree concerning "userfriendly" - I already thought about providing both, e.g. something like this:

("The middle of the nebulous mass in the chest, called φάτνη (phatne, the Manger)")

**Comment by abalkin** [comment]
I would like to focus this issue on the appearance of "primary" names - those that are written next to the star.  With #1647 merged, most of the Almagest stars are given a code label consisting of a 3-letter constellation abbreviation followed by the star index within constellation as it appears in the text of the catalogue, but for 16 stars for which Ptolemy mentions proper names, those names are displayed in preference of a code label.  Most of the Ptolemy names appear in the form "όνομα (Name)" where Ancient Greek name in Greek script is followed by English translation.  A notable exception to this format is Procyon which currently appears as follows: 

![image](https://user-images.githubusercontent.com/535197/122314565-6c514f00-cee6-11eb-99ad-8db79e26b7f9.png)

I suggest shortening "Προκύων (Prokyon, The One [rising] before the Dog)" to "Προκύων (Before the Dog)".

Ideally, I would like to change "όνομα (Name)" format to "Trans (Name)", where Trans is a transliteration of the Greek name, but this would probably require longer discussion.
